recipes/qt/5.x.x/conanfile.py

```diff
@@ -441,6 +441,7 @@ def source(self):
             "-ldbus-1d",
             "-ldbus-1"
         )
+        open(os.path.join(self.source_folder, "qt5", "qtbase", "mkspecs", "features", "uikit", "bitcode.prf"), "w").close()
```

Comment:
Its not obvious how this fixes the issue 😕 if it's only xcode 14 should there be a condition?

//cc @ericLemanissier since maybe he'll be able to help with his divine Qt powers

Reply:
this file contains bitcode flags that are propagated to the build and can't be cancelled with anything else. Simply clearing the file looks easier than throwing in a patch that would basically do the same.

it's not only for Xcode 14, building with bitcode disabled can be done with any earlier Xcode version as well. I mention Xcode 14 because this is the latest toolchain that will be come required soon (in half a year I believe like it was with previous versions) if you want to submit apps to AppStore. And if one doesn't distribute apps via AppStore, there's usually zero reason to enable bitcode.

As soon as Conan has some kind of global bitcode flag that is respected by all recipes, then this will need to be adjusted.

Reply:
There is a safer version with `conan.tools.files.save(os.path.join(self.source_folder, "qt5", "qtbase", "mkspecs", "features", "uikit", "bitcode.prf"), "")`